test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6034,6 +6034,11 @@ def forward(self, x):
                       dynamic_axes={"x": [1, 2]},
                       test_with_inputs=[y])
 
+    def test_prelu_scalar(self):
+        x = torch.scalar_tensor(1.)
+        self.run_test(torch.nn.PReLU(), x, input_names=["x"],
+                      test_with_inputs=[x])
```

Comment:
Remove test_with_inputs. By default `run_test` will test with the inputs in the second argument, so this is redundant.